src/ddscxx/tests/SharedMemory.cpp

```diff
@@ -416,7 +414,7 @@ TYPED_TEST(SharedMemoryTest, loan_sample)
 
   this->SetupCommunication(r_qos, w_qos);
   using DDSType = typename TestFixture::TopicType;
-  // request loan
+  // request loan, only if the type is fixed
```

Comment:
maybe extend the comment and add that it is only supported for fixed-size self-contained types